packages/holocron-module-route/README.md

```diff
@@ -70,15 +76,166 @@ export default function getChildRoutes(store) {
 }
 ```
 
+## 🎛️ API
```

Comment:
A table is broken: https://github.com/americanexpress/holocron/blob/6efd12bf72721be99bc5124fb01f11c2a0e34c72/packages/holocron-module-route/README.md#module-lifecycle-hooks